server/services/licenceService.ts

```diff
@@ -1,20 +1,20 @@
 import type HmppsAuthClient from '../data/hmppsAuthClient'
 import {
+  AdditionalConditionsRequest,
```

Comment:
This was just a format of the file which re-ordered imports to alphabetical order... no real changes